client/modules/IDE/reducers/project.js

```diff
@@ -1,7 +1,10 @@
 import * as ActionTypes from '../../../constants';
+const generate = require('project-name-generator');
+
+const generatedName = generate({ words: 2 }).dashed;
 
 const initialState = {
-  name: 'Hello p5.js'
+  name: `${generatedName}`
```

Comment:
i tried this and found that moving the function call doesn't fix the problem, i think because newProject() doesn't actually create a new project; it just returns the initialState name. this was fine when all new sketches had the same name. hmm, i'll have to think about this a little more... has there been discussion of "untitled" as the sketch name prior to saving? and then the generated name appears on save?